### PR TITLE
changed offlinemode to true

### PR DESCRIPTION
**Pull request by raghuvanshir**
changed offlinemode to true

### CODE REVIEW

**Comment by staff-infrastructure-moj** [comment]
Environment: dev. Security Scan of Stage: dev has failed a IaC security scan, Please look at https://dev.azure.com/MoJ-OFFICIAL/Velocity-Landing-Zone/_build/results?buildId=48776&view=ms.vss-test-web.build-test-results-tab

**Comment by staff-infrastructure-moj** [comment]
Environment: prod. Pipeline Stage: prod
```
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # azurerm_private_dns_txt_record.offlineprint will be updated in-place
  ~ resource "azurerm_private_dns_txt_record" "offlineprint" {
        id                  = "/subscriptions/876eef88-ec3d-47db-b101-ec6f9daefb65/resourceGroups/rg-xwc-dnsresolve-001/providers/Microsoft.Network/privateDnsZones/print.internal.xcp/TXT/offlinemode"
        name                = "offlinemode"
        tags                = {}
        # (4 unchanged attributes hidden)

      - record {
          - value = "FALSE" -> null
        }
      + record {
          + value = "TRUE"
        }
    }

Plan: 0 to add, 1 to change, 0 to destroy.

**Comment by staff-infrastructure-moj** [comment]
Environment: prod. Security Scan of Stage: prod has failed a IaC security scan, Please look at https://dev.azure.com/MoJ-OFFICIAL/Velocity-Landing-Zone/_build/results?buildId=48776&view=ms.vss-test-web.build-test-results-tab

File: README.md [hunk 1]
```diff
@@ -55,8 +55,19 @@ This ensures that making changes in main will make the changes in both environme
...
 
 A [single pipeline](pipelines/deploy-xwc-app.yml) with separate stages for development and prod is used.
 
+## OfflineMode TXT Record
+
+Xerox have a mechanism that assists in users being able to continue printing, but relies on the Xerox client being able to perform a DNS resolution of a TXT record, which depending on the value, indicates the availability of the service.
+
+If the TXT record’s value is ‘True’ then the client will show the cached printers and users can continue to print to cached printers using a standard print queue.
+
+To make it happen via this repo, go to the terraform.tfvars in the respective environment e.g. dev/prod. You will find the variable value_txt_record which is set to FALSE by default.
```
Comment (alancollier142) [inline README.md:69]:
```suggestion
To make it happen via this repo, go to the terraform.tfvars in the respective environment e.g. dev/prod. You will find the variable `value_txt_record` which is set to FALSE by default.
```

**Review by alancollier142 — APPROVED** [review]
Looks good.

**Pull request merged**